frontend/packages/console-app/console-extensions.json

```diff
@@ -304,5 +304,28 @@
         "defaultValue": false
       }
     }
+  },
+  {
+    "type": "console.user-preference/item",
+    "properties": {
+      "id": "console.theme",
+      "label": "%console-app~Theme%",
+      "groupId": "general",
+      "description": "%console-app~Console will appear as per the selected theme.%",
+      "field": {
+        "type": "dropdown",
+        "userSettingsKey": "console.theme",
+        "defaultValue": "systemDefault",
+        "options": [
+          {
+            "value": "systemDefault",
+            "label": "%console-app~System default%"
+          },
+          { "value": "light", "label": "%console-app~Light%" },
+          { "value": "dark", "label": "%console-app~Dark%" }
+        ]
+      },
+      "insertAfter": "console.preferredCreateEditMethod"
```

Comment:
Can we move this new definition before the "Perspectiive" option (`"id": "console.preferredLanguage",`) so that its the first option in User Preferences > General?